alerts/postgres-gke/README.md

```diff
@@ -0,0 +1,31 @@
+# Alerts for PostgreSQL in GKE
+
+## Database size too large alert
+If the database is growing larger than expected (100 GB by default), it may signal excess data being stored or excess records not being properly removed by an application.
```

Comment:
Is it better to be a static value or a percentage? I'm wondering if the static value covers majority of use cases

Reply:
A percentage would be preferable, but there doesn't seem to be a good way to get the limit (max # of bytes available to postgres) to compute the percentage, so we've kept it a static limit.